app/views/pages/home.html.erb

```diff
@@ -69,3 +69,21 @@
     </div> -->
   </div>
 </section>
+<footer role="“contentinfo”">
```

Comment:
Eh. We could. I thought it was only a part of the home page, so I put it here. I think it makes sense for it to be on the other pages, however.